UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -527,6 +548,16 @@
 !endif
 
 
+!if $(SECURE_BOOT_ENABLE) == TRUE
+  gEfiMdeModulePkgTokenSpaceGuid.PcdMaxAuthVariableSize|0x2800
+  gEfiSecurityPkgTokenSpaceGuid.PcdUserPhysicalPresence|TRUE

```

Comment:
Often PcdUserPhysicalPresence should be set to TRUE when user physical presence is detected by a platform driver. 
In this payload, we had better define a MACRO USER_PHYSICAL_PRESENCE for this PCD. This PCD would be set to TRUE only when USER_PHYSICAL_PRESENCE is set to TRUE.